test/t-i18n.test.ts

```diff
@@ -94,23 +91,29 @@ describe("T", () => {
 
 describe("T.$", () => {
```

Comment:
I'd really like to have at least one test with nested XML. There should have already been one, but now that you rewrote large parts of the XML parser, it's all the more important.

Reply:
Added a test for nested XML and a couple others.